Ghidra/Features/Base/src/main/help/help/topics/ExporterPlugin/exporter.htm

```diff
@@ -147,7 +147,7 @@ <H4>Advanced</H4>
                     <TD><I><B>Comment Prefix</B></I><BR>
                     </TD>
 
-                    <TD valign="top">the string to append on the begining of comments<BR>
+                    <TD valign="top">the string to append on the beginning of comments<BR>
```

Comment:
This line would read better as:
`the string to prepend to the beginning of comments`